cookbooks/aws-parallelcluster-shared/attributes/cluster.rb

```diff
@@ -14,6 +14,7 @@
 default['cluster']['change_set_path'] = "#{node['cluster']['shared_dir']}/change-set.json"
 default['cluster']['launch_templates_config_path'] = "#{node['cluster']['shared_dir']}/launch-templates-config.json"
 default['cluster']['instance_types_data_path'] = "#{node['cluster']['shared_dir']}/instance-types-data.json"
+default['cluster']['computefleet_status_path'] = "#{node['cluster']['shared_dir']}/computefleet-status.json"
```

Comment:
Did it end up here because it's used in both `computefleet` and `slurm`? I am afraid there is a dependency between them (`slurm` depends on `computefleet`?) and this attribute should be moved there.

Reply:
Yes, this is used by both computefleet and slurm cookbooks.